### PR TITLE
feature/setWebContentsDebuggingEnabled

### PR DESCRIPTION
**Pull request by hernandazevedo**

### CODE REVIEW

File: QtOffscreenViews/QAndroidOffscreenWebView.cpp [hunk 1]
```diff
@@ -397,6 +397,14 @@ void QAndroidOffscreenWebView::goForward()
 	}
 }
 
+void QAndroidOffscreenWebView::setWebContentsDebuggingEnabled(bool enabled)
+{
+    if (QJniObject * view = offscreenView())
+    {
+        view->callVoid("setWebContentsDebuggingEnabled",(jboolean)enabled);
```
Comment (TheLivingOne) [inline QtOffscreenViews/QAndroidOffscreenWebView.cpp]: `view->callVoid("setWebContentsDebuggingEnabled", static_cast<jboolean>(enabled)); `

File: QtOffscreenViews/QAndroidOffscreenWebView.h [hunk 2]
```diff
@@ -91,6 +91,7 @@ class QAndroidOffscreenWebView
 
 	bool getIgnoreSslErrors() const { return ignore_ssl_errors_; }
 	void setIgnoreSslErrors(bool ignore) { ignore_ssl_errors_ = ignore; }
+    void setWebContentsDebuggingEnabled(bool enabled);
```
Comment (TheLivingOne) [inline QtOffscreenViews/QAndroidOffscreenWebView.h]: Please fix the whitespace (tabs vs spaces, should be as in the rest of the file)

File: QtOffscreenViews/QAndroidOffscreenWebView.cpp (hunk 1 above)
Comment (TheLivingOne) [reply QtOffscreenViews/QAndroidOffscreenWebView.cpp]: The C-style cast now causes a warning in current compilers.

Comment (hernandazevedo) [reply QtOffscreenViews/QAndroidOffscreenWebView.cpp]: Yes, this is fixed on my last commit. Please review.

File: QtOffscreenViews/QAndroidOffscreenWebView.h (hunk 2 above)
Comment (hernandazevedo) [reply QtOffscreenViews/QAndroidOffscreenWebView.h]: Done.

**Review by TheLivingOne — APPROVED** [review]

**Pull request merged**